### PR TITLE
fix to #1365.  Support Java @ModelView classes.

### PR DESCRIPTION
**Pull request by danvwells**
fix to #1365, plus unit test.  @ModelView for java classes' _ViewTable.java would not compile since the column was duplicated.

### CODE REVIEW

File: dbflow-processor/src/main/java/com/raizlabs/android/dbflow/processor/definition/ModelViewDefinition.kt [hunk 1]
```diff
@@ -103,10 +103,6 @@ class ModelViewDefinition(manager: ProcessorManager, element: Element) : BaseTab
                 val columnDefinition = ColumnDefinition(manager, variableElement, this, isPackagePrivateNotInSamePackage)
                 if (columnValidator.validate(manager, columnDefinition)) {
                     columnDefinitions.add(columnDefinition)
-
-                    if (isPackagePrivate) {
-                        columnDefinitions.add(columnDefinition)
```
Comment (agrosner) [inline dbflow-processor/src/main/java/com/raizlabs/android/dbflow/processor/definition/ModelViewDefinition.kt]: This was a mistake and added to wrong field list.

Comment (agrosner) [inline dbflow-processor/src/main/java/com/raizlabs/android/dbflow/processor/definition/ModelViewDefinition.kt]: it should be:
```kotlin
 packagePrivateList.add(columnDefinition)
```

Comment (danvwells) [inline dbflow-processor/src/main/java/com/raizlabs/android/dbflow/processor/definition/ModelViewDefinition.kt]: my pleasure.  I've updated the PR as requested.

**Comment by agrosner** [comment]
awesome thx!

**Pull request merged**